envs/nextflow.yaml

```diff
@@ -4,4 +4,4 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - nextflow
\ No newline at end of file
+  - nextflow
```

Comment:
Could you pin the nextflow version you're using? There are syntax changes between releases that could cause issues.